modules/cas_cache/volume/vol_block_dev_top.c

```diff
@@ -588,7 +588,8 @@ static void _blockdev_set_discard_properties(ocf_cache_t cache,
 		exp_q->limits.discard_granularity =
 			core_q->limits.discard_granularity;
 	} else {
-		blk_queue_max_discard_sectors(exp_q, core_sectors);
+		uint32_t max_discard_size = core_sectors <= UINT_MAX ?: UINT_MAX;
```

Comment:
what does it do? sets max_discard_size to UINT_MAX or 1?

Reply:
I've meant `uint32_t max_discard_size = core_sectors <= UINT_MAX ? core_sectors : UINT_MAX`

Reply:
I see. maybe use MIN macro?

Reply:
done